package.json

```diff
@@ -28,23 +28,28 @@
     "Jonathan Haines <jonno.haines@gmail.com>",
     "Robin Millette <robin@millette.info>"
   ],
+  "types": "types/index.d.ts",
   "files": [
+    "types/index.d.ts",
     "index.js"
   ],
   "dependencies": {
-    "mdast-util-toc": "^4.0.0"
+    "mdast-util-toc": "^5.0.0",
+    "@types/mdast": "^3.0.3"
   },
   "devDependencies": {
     "browserify": "^16.0.0",
+    "dtslint": "^2.0.5",
     "is-hidden": "^1.1.0",
     "negate": "^1.0.0",
     "nyc": "^14.0.0",
-    "prettier": "^1.0.0",
-    "remark": "^11.0.0",
-    "remark-cli": "^7.0.0",
+    "prettier": "^1.19.1",
+    "remark": "^11.0.2",
+    "remark-cli": "^7.0.1",
     "remark-preset-wooorm": "^6.0.0",
-    "tape": "^4.0.0",
+    "tape": "^4.13.0",
     "tinyify": "^2.0.0",
+    "typescript": "^3.7.5",
```

Comment:
I'm not sure a direct dependency on typescript is needed.
DTSLint automatically downloads and runs a bunch of typescript versions for testing.

```suggestion
```